spec/player_spec.rb

```diff
@@ -0,0 +1,44 @@
+require './lib/board'
+require './lib/player'
```

Comment:
This should be require_relative. require should be used for external files, like gems, while require_relative should be used for referring to files within your directory.

Reply:
fixed